server/index.js

```diff
@@ -6,13 +6,31 @@ const port = 6070
 
 app.use(express.json());
 
+let articleTags = '';
+let results = [];
+
 app.get('/api/tips/:id', function(req, res) {
   let restaurantId = req.params.id;
+
   db.getRestaurantInfo(restaurantId, function(error, data) {
     if (error) {
-      console.log('Error at server GET request');
+      console.log('Error at server/restaurants GET request');
+    } else {
+      console.log('Successful server/restaurants GET request');
+      console.log(data);
+      res.send(data);
+    }
```

Comment:
Same comment as before, too many console.logs eventually just fill your console with meaningless information. Also, if I make a request to your api and there is an error, what happens? Does the app crash? As a user how do I know the API is working? Do you send anything back to me letting me know about the _status_ of my request?

Reply:
In this situation, that component simply wouldn't render if there are no relevant articles to display for that particular restaurant. I hadn't thought about sending a message to the user about the request, so I'll definitely keep that in mind going forward! And will be sure to remove those console.logs once everything's closer to being final.